perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -3483,6 +3483,51 @@ public void addAndRemoveMemberInNonSynchronizedGroup() throws Exception {
 		assertTrue("List of members should be empty", members.isEmpty());
 	}
 
+	@Test
+	public void addMemberToGroups() throws Exception {
+		System.out.println(CLASS_NAME + "addMemberToGroups");
+
+		vo = setUpVo();
+
+		groupsManagerBl.createGroup(sess, vo, group);
+		groupsManagerBl.createGroup(sess, vo, group2);
+		groupsManagerBl.createGroup(sess, vo, group3);
+
+		List<Group> groups = Arrays.asList(group, group2, group3);
+
+		Attribute synchroAttr = new Attribute(perun.getAttributesManager().getAttributeDefinition(sess, GroupsManager.GROUPSYNCHROENABLED_ATTRNAME));
+		synchroAttr.setValue("false");
+		perun.getAttributesManager().setAttribute(sess, group, synchroAttr);
+
+		Member member = setUpMember(vo);
+
+		groupsManager.addMember(sess, groups, member);
+
+		for (Group group : groups) {
+			assertTrue("List of members should contain member", groupsManager.isGroupMember(sess, group, member));
+		}
+	}
+
+
+	@Test
+	public void addMembersToGroup() throws Exception {
+		System.out.println(CLASS_NAME + "addMembersToGroup");
+
+		vo = setUpVo();
+		setUpGroup(vo);
+
+		Attribute synchroAttr = new Attribute(perun.getAttributesManager().getAttributeDefinition(sess, GroupsManager.GROUPSYNCHROENABLED_ATTRNAME));
```

Comment:
Same as in previous test. Method logic (and test) should work even if attribute is not set at all.